Products/ZenModel/Device.py

```diff
@@ -2353,8 +2372,6 @@ def _getPingStatus(self, statusclass):
         if not self.zPingMonitorIgnore and self.getManageIp():
             from Products.Zuul import getFacade
             from Products.ZenEvents.events2.proxy import EventProxy
-            from zenoss.protocols.protobufs.zep_pb2 import STATUS_NEW, STATUS_ACKNOWLEDGED, \
```

Comment:
You can't remove these local imports because they're used locally. You could move them to the module scope instead.